app/code/Magento/Multishipping/view/frontend/templates/checkout/link.phtml

```diff
@@ -5,4 +5,4 @@
  */
 
 ?>
```

Comment:
You missed defining phpdoc block for escaper here

Reply:
fixed